@xen-orchestra/xapi/host/_ipmi.mjs

```diff
@@ -25,7 +25,27 @@ export const IPMI_SENSOR_REGEX_BY_DATA_TYPE_BY_SUPPORTED_PRODUCT_NAME = {
     [IPMI_SENSOR_DATA_TYPE.psuPower]: /^psu[0-9]+_pin$/i,
     [IPMI_SENSOR_DATA_TYPE.ip]: /^ip address$/i,
   },
+  dell: {
+    [IPMI_SENSOR_DATA_TYPE.fanSpeed]: /^fan[0-9]+(a|b)$/i,
+    [IPMI_SENSOR_DATA_TYPE.psuPower]: /^voltage [0-9]+$/i,
+    [IPMI_SENSOR_DATA_TYPE.psuStatus]: /^ps[0-9]+ pg fail$/i,
+    [IPMI_SENSOR_DATA_TYPE.cpuTemp]: /^temp$/i,
+    [IPMI_SENSOR_DATA_TYPE.totalPower]: /^pwr consumption$/i,
+    [IPMI_SENSOR_DATA_TYPE.inletTemp]: /^inlet temp$/i,
+    [IPMI_SENSOR_DATA_TYPE.ip]: /^ip address$/i,
+  },
```

Comment:
I really think this should be in a config file , at least when we will want to add another support . Proposed format : 
{
   test: <regexp>, 
   label:string,
   sensors: {
       [type]=> regexp
      }
      }

Reply:
This was suggested during a call with Olivier, but as Olivier mentioned, the plugin will likely disappear...
So, I therefore prefer to limit the number of changes, since the code will likely be removed later...